sdks/python/apache_beam/testing/load_tests/build.gradle

```diff
@@ -29,16 +29,14 @@ def loadTestArgsProperty = "loadTest.args"
 
 def runnerProperty = "runner"
 
-task run(type: Exec) {
-    dependsOn 'setupVirtualenv'
-    dependsOn 'installGcpTest'
-
+task run(type: Exec, dependsOn: installGcpTest) {
     def loadTestArgs = project.findProperty(loadTestArgsProperty) ?: ""
 
     def runnerArg = project.findProperty(runnerProperty) ?: ""
     if (runnerArg == 'DataflowRunner' || runnerArg == 'TestDataflowRunner') {
-        dependsOn 'sdist'
-        loadTestArgs +=" --sdk_location=${project.buildDir}/apache-beam.tar.gz"
+        def sdistPath = "${project.rootDir}/sdks/python/apache_beam/testing/load_tests/build/apache-beam.tar.gz"
+        dependsOn "sdist"
+        loadTestArgs +=" --sdk_location=${sdistPath}"
```

Comment:
If `${sdistPath}` is the same as `${project.buildDir}/apache-beam.tar.gz`, then please revert to minimize diffs and hard-coding project directory structure.

Reply:
You are right. I don't know why in past `buildDir` wasn't pointing this module build dir but now is ok. Thank you, I changed it